samples/msgext-northwind-inventory-csharp/AdaptiveCards/successCard.json

```diff
@@ -0,0 +1,254 @@
+                                            "value": "${string(unitsInStock)}"
+                                        },                                      
+                                        {
+                                            "title": "Units on order",
+                                            "value": "${string(unitsOnOrder)}"
+                                        },
+                                        {
+                                            "title": "Reorder Level",
+                                            "value": "${string(reorderLevel)}"
+                                        },
+                                        {
+                                            "title": "Revenue this period",
+                                            "value": "${string(revenue)} USD"
+                                        }
+                                    ],
+                                    "separator": true
+                                }
+                            ]
+                        }
+                     
+
+          ]
+        }]
+    },
+    {
+      "type": "Container",
+      "items": [
+        {
+          "type": "ActionSet",
+          "actions": [
+            {
+              "type": "Action.ShowCard",
+              "title": "Take action",
+              "card": {
+                "type": "AdaptiveCard",
+                "body": [
+                  {
+                    "type": "Input.Text",
+                    "id": "txtStock",
+                    "label": "Quantity",
+                    "min": 0,
+                    "max": 9999,
+                    "errorMessage": "Invalid input, use whole positive number",
+                    "style": "Tel"
+                  }
+                ],
+                "actions": [
+                  {
+                    "type": "Action.Execute",
+                    "title": "Update stock ✅",
+                    "verb": "ok",
+                    "data": {
+                      "productId": "${productId}"
+                    }
+                  },
+                  {
+                    "type": "Action.Execute",
+                    "title": "Restock 📦",
+                    "verb": "restock",
+                    "data": {
+                      "productId": "${productId}"
+                    }
+                  },
+                  {
+                    "type": "Action.Execute",
+                    "title": "Cancel restock ❌",
+                    "verb": "cancel",
+                    "data": {
+                      "productId": "${productId}"
+                    }
+                  },
+                  {
+                      "type": "Action.OpenUrl",
+                      "title": "Handoff to bot",
+                      "url": "https://teams.microsoft.com/l/chat/0/0?users=28:758dcadb-01d6-4b75-a8af-3c19d8e84687&continuation=test-continuation-token"
```

Comment:
@Jegadeesh-MSFT - Please remove these hardcoded values.

Reply:
I have fixed this.